### DISCUSSION TITLE
Linux Mint 22 Wilma Themes / Design Issue ??

### DISCUSSION THREAD

**Discussion by BCStarfighter** [linuxmint/discussions#611] [Questions]
Hello, 

i have a problem with the Themes / Design Manager. Iḿ Linux Beginner come form Windows. I Install new Linux Mint 22 Wilma with all Updates. I install also my VPN Proton. I use for this the official guide on the proton website. After this i cannot open unter Settings the Desk with the Themes / Design Options. It comes a message a Issue it happens... Can someone help me to fix this problem ? 
I try to reset the Display Manager gdm3 and reboot but nothing works. I try also to change another desktop theme by login but it not display antoher option. I reinstall also the cinamon desktop theme in the terminal but nothing works ??

**Comment by JosephMcc** [comment]
You'd be much more likely to get help on the Mint forums. This isn't really the place for that.

**Comment by mtwebster** [comment]
https://forums.linuxmint.com/
